chain/epoch-manager/src/genesis.rs

```diff
@@ -163,7 +162,7 @@ pub(crate) fn find_threshold(
 ) -> Result<Balance, EpochError> {
     let stake_sum: Balance =
         stakes.iter().fold(Balance::ZERO, |sum, item| sum.checked_add(*item).unwrap());
-    if stake_sum.as_yoctonear() < num_seats.into() {
+    if stake_sum < Balance::from_yoctonear(u128::from(num_seats)) {
```

Comment:
Extract this conversion to balance into a variable with a semantically meaningful name. Perhaps something like `min_possible_stake`?